src/pages/holofuel/Inbox/Inbox.test.js

```diff
@@ -0,0 +1,179 @@
+import React from 'react'
+import { render, fireEvent, act, within } from '@testing-library/react'
+import wait from 'waait'
+import { ApolloProvider } from '@apollo/react-hooks'
+import { MockedProvider } from '@apollo/react-testing'
+import apolloClient from 'apolloClient'
+import Inbox, { TransactionRow } from './Inbox'
+import { pendingList } from 'mock-dnas/holofuel'
+import { TYPE } from 'models/Transaction'
+import HolofuelOfferMutation from 'graphql/HolofuelOfferMutation.gql'
+import HolofuelAcceptOfferMutation from 'graphql/HolofuelAcceptOfferMutation.gql'
+import HolofuelActionableTransactionsQuery from 'graphql/HolofuelActionableTransactionsQuery.gql'
+
+const actionableTransactions = pendingList.requests.concat(pendingList.promises).reverse().map(item => {
+  if (item.event[2].Request) {
+    return item.event[2].Request
+  } else if (item.event[2].Promise) {
+    return item.event[2].Promise.tx
+  } else {
+    throw new Error('unrecognized transaction type', item.toString())
+  }
+})
+
+jest.mock('data-interfaces/EnvoyInterface')
+// mocking Header because it depends on Router
+jest.mock('components/holofuel/Header')
+
+describe('Inbox Connected', () => {
+  it('renders', async () => {
+    let getAllByRole
+
+    await act(async () => {
+      ({ getAllByRole } = render(<ApolloProvider client={apolloClient}>
+        <Inbox />
+      </ApolloProvider>))
+      await wait(15)
+    })
+
+    const listItems = getAllByRole('listitem')
+    expect(listItems).toHaveLength(2)
+
+    listItems.forEach((item, index) => {
+      const { getByText } = within(item)
+      expect(getByText(actionableTransactions[index].notes)).toBeInTheDocument()
```

Comment:
and I think this is an acceptable way to test if something is in your row, without having to do the extra layer of `within()` that we used in the tests for your components.

No need to update, just wanted to keep you posted on what I'm thinking re RTL.